### PR TITLE
New release v3.0.0

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

**Comment by holgerd77** [comment]
Also did a quick VM check on this (by injecting into `node_modules` folder, running some tests and adopting failing paths). Worked as expected.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+```javascript
+let Account = require('ethereumjs-account').default
+```
+
+The library now also comes with a **type declaration file** distributed along with the package published.
+
+- Migration of code base and toolchain to `TypeScript`, PR [#27](https://github.com/ethereumjs/ethereumjs-account/pull/27)
+- Updated `ethereumjs-util` dependency, PR [#25](https://github.com/ethereumjs/ethereumjs-account/pull/25)
+- Updated `standard`, `nyc` dev dependencies, PRs [#24](https://github.com/ethereumjs/ethereumjs-account/pull/24), [#22](https://github.com/ethereumjs/ethereumjs-account/pull/22)
```
Comment (s1na) [inline CHANGELOG.md]: It seems these two PRs (also #25) have been closed and not merged. The `standard` dependency has been in fact dropped in #27.

**Review by s1na — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (holgerd77) [reply CHANGELOG.md]: Ah, thanks a lot for catching this, one really has to think also on doing release notes. Hmm.

Have updated and force-pushed.

**Review by s1na — APPROVED** [review]

**Comment by holgerd77** [comment]
Just wanted to publish with `npm`, but release breaks locally for me with:

```shell
/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/test/index.ts:11
      account.stateRoot.toString('hex'),
                        ^
TypeError: Cannot read property 'toString' of undefined
    at Test.<anonymous> (/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/test/index.ts:11:25)
    at Test.bound [as _cb] (/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/node_modules/tape/lib/test.js:64:32)
    at Test.run (/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/node_modules/tape/lib/test.js:83:10)
    at Test.bound [as run] (/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/node_modules/tape/lib/test.js:64:32)
    at Test._end (/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/node_modules/tape/lib/test.js:152:11)
    at Test.bound [as _end] (/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/node_modules/tape/lib/test.js:64:32)
    at Immediate._onImmediate (/Users/hdrewes/Documents/DEV/EthereumJS/ethereumjs-account/node_modules/tape/lib/test.js:105:18)
    at runCallback (timers.js:810:20)
    at tryOnImmediate (timers.js:768:5)
    at processImmediate [as _immediateCallback] (timers.js:745:5)
```

Can you confirm that? No idea yet what's going on and why this didn't happen on the CI tests?

Hmm.

**Comment by s1na** [comment]
Not sure why that's happened, I haven't been able to reproduce it. This happens when running `npm publish`?

**Comment by holgerd77** [comment]
Ah, was just some dependency installation thing, reinstalled `node_modules`, worked now. Just published.

**Pull request merged**